roles/kubernetes/client/tasks/main.yml

```diff
@@ -95,6 +95,7 @@
     dest: "{{ artifacts_dir }}/kubectl"
   become: no
   run_once: yes
+  delegate_to: localhost
```

Comment:
The original logic works. This will try to sync from localhost to localhost

Reply:
It's from this https://github.com/kubernetes-sigs/kubespray/pull/5117

Reply:
well okay